### PR TITLE
When remote input is disabled, also disable clipboard management

### PR DESCRIPTION
**Pull request by yobert**
Hello! I think a good feature would be that when remote input is disabled, also disable the clipboard function.

My reasoning is that a common use case for input being disabled is you are showing your screen to a large or untrusted audience, and many password managers use the clipboard as a medium for passwords. Someone might not realize that even though they have remote input disabled, their password may become visible to a guest who pulls up the clipboard tool.

Please forgive my implementation--- I am definitely not a C++ expert. Also thank you for making such awesome software.

### CODE REVIEW

File: src/data-control.c [hunk 1]
```diff
@@ -224,6 +224,10 @@ struct zwlr_data_control_source_v1_listener data_control_source_listener = {
 };
 
 static struct zwlr_data_control_source_v1* set_selection(struct data_control* self, bool primary) {
+	if(!self->manager) {
```
Comment (any1) [inline src/data-control.c:227]: This doesn't do anything if `data_control` isn't initialised.

Comment (yobert) [reply src/data-control.c]: Cool! Okay I took out that check and force pushed to update the PR.

Comment (yobert) [reply src/data-control.c]: (also tested and it seems good!)

**Comment by any1** [comment]
Thanks!

**Pull request merged**